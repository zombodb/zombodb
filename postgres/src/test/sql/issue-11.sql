BEGIN;

CREATE SCHEMA issue_11;
CREATE TABLE issue_11.test_table ();
SELECT count_of_table('issue_11.test_table');

ABORT;