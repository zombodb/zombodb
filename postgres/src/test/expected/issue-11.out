BEGIN;
CREATE SCHEMA issue_11;
CREATE TABLE issue_11.test_table ();
SELECT count_of_table('issue_11.test_table');
 count_of_table 
----------------
              0
(1 row)

ABORT;
